### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by I11ia**
- [DEMO LINK](https://I11ia.github.io/layout_catalog/)
- [TEST REPORT LINK](https://I11ia.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/main-cards.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.cards {
```
Comment (TarasHoliuk) [inline src/styles/blocks/main-cards.scss:1]: let's rename the file to cards.scss to match BEM-block name

File: src/index.html [hunk 2]
```diff
@@ -4,9 +4,368 @@
...
+            <a class="nav__link" href="smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops & computers"
```
Comment (TarasHoliuk) [inline src/index.html:46]:
```suggestion
              href="#laptops_and_computers"
```
don't use spaces and `&` in anchor links

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Almost perfect
let's fix a few minor things

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**